e2e/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apk add --no-cache --virtual .build-deps bash gcc musl-dev openssl go git aw
 RUN go get github.com/onsi/ginkgo/ginkgo && go get github.com/onsi/gomega/...
 
 # Copy the binary
-ADD bin/local/copilot-amd64 /bin/copilot
+ADD bin/local/copilot-linux-amd64 /bin/copilot
```

Comment:
Because in Makefile [compile-linux ](https://github.com/aws/copilot-cli/blob/def8fe93f2e2fbe902a335197c2647e065acf64b/Makefile#L47
) the destination is actually copilot-linux-amd64, not copilot-amd64. I guess the Makefile was updated at some point but the Dockerfile wasn't